pkg/scyllaclient/client_scylla.go

```diff
@@ -906,7 +907,7 @@ func (c *Client) LoadSSTables(ctx context.Context, host, keyspace, table string,
 	const WIPError = "Already loading SSTables"
 
 	_, err := c.scyllaOps.StorageServiceSstablesByKeyspacePost(&operations.StorageServiceSstablesByKeyspacePostParams{
-		Context:            forceHost(ctx, host),
+		Context:            noTimeout(forceHost(ctx, host)),
```

Comment:
Please don't use noTimeout.
Use customTimeout instead and set 30m to not hang forever.